dev-infrastructure/modules/aks-cluster-base.bicep

```diff
@@ -460,6 +462,82 @@ resource aksCluster 'Microsoft.ContainerService/managedClusters@2024-04-02-previ
   ]
 }
 
+resource aksDiagnosticSettings 'Microsoft.Insights/diagnosticSettings@2017-05-01-preview' = if (logAnalyticsWorkspaceId != '') {
+  scope: aksCluster
+  name: aksClusterName
+  properties: {
+    logs: [
+      {
+        category: 'kube-audit'
+        enabled: true
+      }
+      {
+        category: 'kube-audit-admin'
+        enabled: true
+      }
+    ]
+    workspaceId: logAnalyticsWorkspaceId
+  }
+}
+
+resource aksClusterDcr 'Microsoft.Insights/dataCollectionRules@2023-03-11' = if (logAnalyticsWorkspaceId != '') {
```

Comment:
There is already a DCR in place [here](https://github.com/Azure/ARO-HCP/blob/main/dev-infrastructure/modules/metrics/datacollection.bicep) which I _think_ we could add this configuration to.  Are any downsides to having multiple DCRs configured on cluster?

Reply:
I thought different dcr for different purpose. I don't think it matters from a technical standpoint whether we have one or two. I also could not tell if the name of the existing one (MSProm-...) is intentional or not and I opted to avoid renaming/re-creating existing resources.